grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -269,7 +269,7 @@ class UrlMappings {
         }
 
         "/api/stockMovements/importPickListItems/$id"(parseRequest: true) {
```

Comment:
Change the URL mapping to 
```
/picklists/importCsv 
```
or let the default URL mapping configuration handle it.

Reply:
I also need ot indlude stockMvoement id in the mapping so a simple `/picklist/importCsv` needs to somehow include which stockMovement are we importing it to.

I suggest leaving the mapping as is, since also we would have to deprecate the old mapping as well before moving it to the new URL

Reply:
I was thinking the ID could go into the multipart request data. But I'm going with that for now. 

Just make the URL match the other RPC endpoints on Stock Movement so it's clear what we're doing. And rename the action to "importPicklist" to match the client and what the service method should be called.
```
/api/stockMovements/$id/importPicklist
```